pkg/server/server.go

```diff
@@ -1156,6 +1156,7 @@ func (s *Server) resolveTypesystem(ctx context.Context, storeID, modelID string)
 
 	typesys, err := s.typesystemResolver(ctx, storeID, modelID)
 	if err != nil {
+		telemetry.TraceError(span, err)
```

Comment:
arguably this should be in the `main` branch, I can send a separate PR

Reply:
Should it? The convention seems to be tracing a function's returned error but not necessarily in the function itself.

Reply:
Not sure what you mean? This function itself is already creating a span, so if it errors out we should mark it as errored.